code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -288,11 +288,42 @@ USE THIS CHEMISTRY DISPENSER FOR MAPS SO THEY START AT 100 ENERGY
 	ui_interact(user)
 //Cafe stuff
 
-/obj/machinery/chem_dispenser/brewer/
+/obj/machinery/chem_dispenser/nonmedical/

```

Comment:
Paths are actually the bad way of doing this.
You should be making a can_use(robot) proc and using that instead in the base attackby() code for dispensers.
That's OOP.